assets/gql/triggers/check_warning.gql

```diff
@@ -0,0 +1,9 @@
+#import "./fields.frag.gql"
+
+mutation CheckTriggerWarnings($input: TriggerInput!) {
+  checkTriggerWarnings(input: $input) {
+    ...TriggerFields
+
+    ...ErrorFields
```

Comment:
The response format should be the same as publishFlow API
An attribute for success and errors. We don't need the trigger values here